Challenge3/src/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Finding, FindingSeverity, FindingType, ethers, Alert, AlertsResponse, getAlerts } from "forta-agent";
+import { L2_DAI_TOKEN_ADDRESS, L2_TOKEN_ABI } from "./constants";
+import { Provider } from "@ethersproject/providers";
+
+export type GetAlertsFunction = (options: {
```

Comment:
Instead of declaring this, you can directly import `GetAlerts` type from the SDK